5_databases/week1/homework-lesson1.sql

```diff
@@ -0,0 +1,45 @@
+-- Active: 1706443592393@@127.0.0.1@3306@hyf
+
+--Find out how many tasks are in the task table
+SELECT COUNT(task.id) AS total_tasks FROM task
+
+--Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(task.id) FROM task
+WHERE task.due_date is NULL
+
+--Find all the tasks that are marked as done
+SELECT task.title, status.name AS task_done
+ FROM (task JOIN status ON task.status_id=status.id)
+ WHERE status.name = 'Done';
+
+--Find all the tasks that are not marked as done
+SELECT task.title, status.name AS task_done
+ FROM (task JOIN status ON task.status_id=status.id)
+ WHERE status.name != 'Done';
+
+--Get all the tasks, sorted with the most recently created first
+SELECT task.created AS created_task from task ORDER BY created DESC
```

Comment:
This is the right query however you are only showing the dates. can you also show the task info?